docs/source/en/quantization/overview.md

```diff
@@ -45,32 +45,56 @@ In short, supporting a wide range of quantization methods allows you to pick the
+| [FBGEMM_FP8](./fbgemm_fp8.md)              | 🟢                       | 🔴              | 🟢        | 🔴              | 🔴                                 | 🔴              | 🔴                        | 8              | 🔴                                   | 🟢            | 🟢                      | https://github.com/pytorch/FBGEMM       |
+| [torchao](./torchao.md)                    | 🟢                       |                 | 🟢        | 🔴              | 🟡 <sup>5</sup> | 🔴              |                       | 4 / 8          |                                   | 🟢🔴           | 🟢                      | https://github.com/pytorch/ao       |
+| [VPTQ](./vptq.md)                             | 🔴                       | 🔴              |     🟢     | 🟡              | 🔴                                 | 🔴              | 🟢                      | 1 / 8          | 🔴                                   | 🟢            | 🟢                      | https://github.com/microsoft/VPTQ            |
 
 <Tip>
+  
+**<sup>1</sup>** bitsandbytes is being refactored to support multiple backends beyond CUDA. Currently, ROCm (AMD GPU) and Intel CPU implementations are mature, with Intel XPU in progress and Apple Silicon support expected by Q4/Q1. For installation instructions and the latest backend updates, visit [this link](https://huggingface.co/docs/bitsandbytes/main/en/installation#multi-backend). Check out [these docs](https://huggingface.co/docs/bitsandbytes/main/en/non_cuda_backends) for more details and feedback links.
 
-\* bitsandbytes is being refactored to support multiple backends beyond CUDA. Currently, ROCm (AMD GPU) and Intel CPU implementations are mature, with Intel XPU in progress and Apple Silicon support expected by Q4/Q1. For installation instructions and the latest backend updates, visit [this link](https://huggingface.co/docs/bitsandbytes/main/en/installation#multi-backend).
+</Tip>
+
+<Tip>
+
+**<sup>2</sup>** bitsandbytes is seeking contributors to help develop and lead the Apple Silicon backend. Interested? Contact them directly via their repo. Stipends may be available through sponsorships.
+
+</Tip>
+
+<Tip>
+
+**<sup>3</sup>** GPTQModel[CPU] supports full bit range via Torch and 4-bit via IPEX on Intel/AMD.
 
-We value your feedback to help identify bugs before the full release! Check out [these docs](https://huggingface.co/docs/bitsandbytes/main/en/non_cuda_backends) for more details and feedback links.
+</Tip>
+
+<Tip>
+
+**<sup>4</sup>** GPTQModel[Intel GPU] via IPEX only supports 4-bit for Intel Datacenter Max + Arc.
 
 </Tip>
 
 <Tip>
 
-\** bitsandbytes is seeking contributors to help develop and lead the Apple Silicon backend. Interested? Contact them directly via their repo. Stipends may be available through sponsorships.
+**<sup>5</sup>** torchao only supports int4 weight on Metal (Apple Silicon).
+
+</Tip>
+
+
+<TIP>
+
+**<sup>6</sup>** [See GGUF section](../gguf.md)
 
-</Tip>
\ No newline at end of file
+</TIP>
```

Comment:
For GGUF, can you revert the changes you did ?

Reply:
@SunMarc  Done.  Please check But I also made following clean-ups to the table. The table is horizontally overflowing on my 14.5 inch 16x10 screen even with the edits. Something needs to be done and break the table into single-view digestible version.

* Fixed using subscript (instead of superscript) notation for the footnotes. 
* Renamed some of the column headers for brevity so we can fit more columns without overflow. See following.
* `Just in time Quantization` => `Runtime Quantization`
* `RoCm GPU (AMD)` => `ROCm GPU` 
* `Fine-tuning (via PEFT)` => `PEFT Fine Tuning`

Reply:
@SunMarc 

Our next PR will also address the issue of GPTQConfig `__init__` forcing pass `bits`. It is actually not required and hinders api brevity when launching inference only mode using `backend="marlin"`.  Add this here so I won't forget. 


```py
AutoModelForCausalLM.from_pretrained(
    "ModelCloud/Opt-125-GPTQ-4bit-10-25-2024", device_map="auto", 
     quantization_config=GPTQConfig(bits=4, backend="marlin")) 
```

Can be optimized to:

```py
AutoModelForCausalLM.from_pretrained(
    "ModelCloud/Opt-125-GPTQ-4bit-10-25-2024", device_map="auto", 
     quantization_config=GPTQConfig(backend="marlin")) 
```

Reply:
Perfect thanks !

Reply:
@SunMarc  Done. As far as other columns, I can't find shorter alternatives. `transformers` is pretty long single word. 

Fixed table issue where bits values are wrapping due to spacing between `1  / 2 / 4 / 8`, now they are `1/2/4/8` so they don't wrap to next line. 

https://github.com/huggingface/transformers/pull/35012/commits/945f6633dea723613e4961c20c69b2a1f92f1b4f